frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -6,184 +6,213 @@
-      "typeDescription": "%kubevirt-plugin~**Virtual Machines** are templates for quickly creating a virtual machine.%",
-      "filters": [
-        {
-          "label": "%kubevirt-plugin~Template Providers%",
-          "attribute": "templateProvider"
-        }
-      ]
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1alpha3"
+      },
+      "component": { "$codeRef": "VirtualizationPage.VirtualizationPage" }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1alpha3_API"]
     }
   },
   {
-    "type": "console.catalog/item-provider",
+    "type": "console.navigation/resource-ns",
     "properties": {
-      "catalogId": "dev-catalog",
-      "type": "VmTemplate",
-      "provider": { "$codeRef": "createVM.catalogVMTemplateProvider" }
+      "id": "virtualmachines",
+      "section": "virtualization",
+      "name": "%kubevirt-plugin~VirtualMachines%",
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1"
+      }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1_API"]
     }
   },
   {
-    "type": "console.catalog/item-filter",
+    "type": "console.navigation/resource-ns",
     "properties": {
-      "catalogId": "dev-catalog",
-      "type": "Template",
-      "filter": { "$codeRef": "createVM.catalogVMTemplateFilter" }
+      "id": "virtualmachines",
+      "section": "virtualization",
+      "name": "%kubevirt-plugin~VirtualMachines%",
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1alpha3"
+      }
     },
     "flags": {
-      "required": ["KUBEVIRT"]
+      "required": ["KUBEVIRT", "KUBEVIRT_HAS_v1alpha3_API"]
     }
   },
   {
-    "type": "console.redux-reducer",
+    "type": "console.page/resource/details",
     "properties": {
-      "scope": "kubevirt",
-      "reducer": { "$codeRef": "reduxReducer" }
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine"
+      },
+      "component": { "$codeRef": "VirtualMachinesDetailsPage.VirtualMachinesDetailsPage" }
     },
     "flags": {
       "required": ["KUBEVIRT"]
     }
   },
   {
-    "type": "console.context-provider",
+    "type": "console.navigation/href",
     "properties": {
-      "provider": { "$codeRef": "contextProvider.CDIUploadProvider" },
-      "useValueHook": { "$codeRef": "contextProvider.useCDIUploadHook" }
+      "id": "virtualmachinetemplates",
+      "section": "virtualization",
+      "perspective": "admin",
+      "name": "%kubevirt-plugin~Templates%",
+      "href": "/virtualmachinetemplates",
+      "prefixNamespaced": true
     },
     "flags": {
       "required": ["KUBEVIRT"]
     }
   },
   {
-    "type": "console.page/route/standalone",
+    "type": "console.page/route",
```

Comment:
should use the list page
so the route for the list page:
[route to list page] - show list
[route to list page]/~new - show new object page (yaml)
[route to list page]/~new/form - show new object page (form a.k.a wizard)